common/djangoapps/track/shim.py

```diff
@@ -113,6 +117,9 @@ def __call__(self, event):
             return
         payload = event['event']
 
+        if isinstance(payload, basestring):
+            return
```

Comment:
This is kind of moot, since I removed this code. However - as an aside, the mobile app emits video events that will have normal dict payloads. I was able to remove this code because there is a check higher up in the function that ensures that only certain event names are allowed to continue through the shim. Those event types are only emitted by the mobile app, so at this point in the code we will only be seeing dict payloads.